Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -15,6 +15,15 @@ open class _ChatChannelListItemView<ExtraData: ExtraDataTypes>: _ChatChannelSwip
     public var content: (channel: _ChatChannel<ExtraData>?, currentUserId: UserId?) {
         didSet { updateContentIfNeeded() }
     }
+    
+    /// Properties tied to `ChatChannelListItemView` layout
+    public struct Layout {
+        public fileprivate(set) var timestampLabelConstraints: [NSLayoutConstraint] = []
+        public fileprivate(set) var avatarViewConstraints: [NSLayoutConstraint] = []
+        public fileprivate(set) var titleLabelConstraints: [NSLayoutConstraint] = []
+        public fileprivate(set) var subtitleLabelConstraints: [NSLayoutConstraint] = []
+        public fileprivate(set) var unreadCountViewConstraints: [NSLayoutConstraint] = []
```

Comment:
Even though it seems obvious from the name of the variable, it should still be documented properly.

Reply:
eg. "Constraints of `unreadCountView`" or something different / more detailed?

Reply:
Yep, something like that. We have this rule internally to prevent discussions if something is worth/needed to be documented. This way you don't have to think about it - is it `public` -> should be documented.